packages/web/src/features/languages/ManageLanguageView.tsx

```diff
@@ -138,7 +138,14 @@ export default function ManageLanguageView() {
   const removeMemberMutation = useRemoveLanguageMemberMutation();
   const updateMemberMutation = useUpdateLanguageMemberMutation();
 
-  const formContext = useForm<FormData>();
+  const formContext = useForm<FormData>({
+    defaultValues: {
+      name: language.data.name,
+      font: language.data.font,
+      textDirection: language.data.textDirection,
+      bibleTranslationIds: language.data.bibleTranslationIds,
+    },
+  });
```

Comment:
We can specify default values in one place using `useForm`. React hook form will ensure that each input is set correctly from this